learning_journal_db/tests.py

```diff
@@ -72,98 +79,81 @@ def dummy_http_request(new_session):
     return test_request
 
 
-def dummy_http_request_post(title, body, date, new_session):
+def dummy_http_request_post(title, body, new_session):
     """Define a dummy request with method="POST"."""
     test_request = testing.DummyRequest()
     test_request.dbsession = new_session
     test_request.method = 'POST'
     test_request.POST['title'] = title
     test_request.POST['body'] = body
-    test_request.POST['date'] = date
     return test_request
 
 
-def test_lists_title(new_session):
-    """Test whether lists() pull out correct info from db."""
-    from .views.default import lists
-    new_session.add(MyModel(title='test1', body='test2', date='test3'))
-    new_session.flush()
-    result = lists(dummy_http_request(new_session))
-    for entry in result['entries']:
-        assert entry.title == 'test1'
-
-
-def test_lists_body(new_session):
-    """Test whether lists() pull out correct info from db."""
-    from .views.default import lists
-    new_session.add(MyModel(title='test_1', body='test2', date='test3'))
-    new_session.flush()
-    result = lists(dummy_http_request(new_session))
-    for entry in result['entries']:
-        assert entry.body == 'test2'
+ATTR_VAL = [
+    ('title', 'test1'),
+    ('body', 'test2')
+]
 
 
-def test_lists_date(new_session):
+@pytest.mark.parametrize('attr, val', ATTR_VAL)
+def test_lists(attr, val, new_session):
     """Test whether lists() pull out correct info from db."""
-    from .views.default import lists
-    new_session.add(MyModel(title='test1', body='test2', date='test3'))
+    new_session.add(MyModel(title='test1', body='test2'))
     new_session.flush()
     result = lists(dummy_http_request(new_session))
     for entry in result['entries']:
-        assert entry.date == 'test3'
+        assert getattr(entry, attr) == val
```

Comment:
Can you use the `attr` and `val` to get your entry written?  Then you could write a few different entries and verify that the values you'd requested were properly entered.